package.json

```diff
@@ -10,8 +10,7 @@
     "typescript": "^4.8.4"
   },
   "scripts": {
-    "cypress": "cypress open",
```

Comment:
We should not have changed anything in the package.json

Reply:
Thank you for noticing this change. I wasn't aware of this update.